functions/v2/pubsub/requirements.txt

```diff
@@ -0,0 +1 @@
+google-cloud-pubsub==2.7.0
```

Comment:
We shouldn't need this 🙂

Reply:
True, if we eliminate the first part of the sample. Will we do that for all languages?

Reply:
Looks like only Nodejs had this, and it was eliminated in the new sample. So removing this.